packages/cubejs-snowflake-driver/src/SnowflakeDriver.ts

```diff
@@ -451,6 +454,11 @@ export class SnowflakeDriver extends BaseDriver implements DriverInterface {
       await this.execute(connection, 'ALTER SESSION SET TIMEZONE = \'UTC\'', [], false);
       await this.execute(connection, `ALTER SESSION SET STATEMENT_TIMEOUT_IN_SECONDS = ${this.config.executionTimeout}`, [], false);
 
+      // We only want to ignore the case if someone sets the value to false explicitly since the default assumption
+      // is that casing matters
+      if (!this.config.identIgnoreCase) {
+        await this.execute(connection, 'ALTER SESSION SET QUOTED_IDENTIFIERS_IGNORE_CASE = FALSE', [], false);
```

Comment:
default is `FALSE` https://docs.snowflake.com/en/sql-reference/identifiers-syntax so I think it should be:
```
  if (this.config.identIgnoreCase) {
        await this.execute(connection, 'ALTER SESSION SET QUOTED_IDENTIFIERS_IGNORE_CASE = TRUE', [], false);
}
        ```